.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+### Contribution guidelines:
+
+- Be polite.
+- Branch naming convention: `{ IssueID }-short-description`.
+- Commit naming convention: `{ IssueID } { Issue Type (e.g. Enhancement, Bug, etc.) } short description`
+- Run `npm test`
+- Run `xo --fix` if you have any [coding style](https://github.com/sindresorhus/xo) issues. They will be automatically fixed for you.
+
+##### When opening a pull request:
+1. Make sure you squashed all your committed into one. A clean commit history is important.
```

Comment:
what if someone releases a bigger refactoring? When having bigger changes it's usually better to have logical chunks, so enforcing squashes for any PR is IMHO not the best solution